test/integration/opaqueports/opaque_ports_test.go

```diff
@@ -15,16 +15,24 @@ import (
 var TestHelper *testutil.TestHelper
 
 var (
-	opaquePodApp = "opaque-pod"
-	opaquePodSC  = "slow-cooker-opaque-pod"
-	opaqueSvcApp = "opaque-service"
-	opaqueSvcSC  = "slow-cooker-opaque-service"
-	tcpMetricRE  = regexp.MustCompile(
+	opaquePodApp         = "opaque-pod"
+	opaquePodSC          = "slow-cooker-opaque-pod"
+	opaqueSvcApp         = "opaque-service"
+	opaqueSvcSC          = "slow-cooker-opaque-service"
+	opaqueUnmeshedSvcPod = "opaque-unmeshed-svc"
+	opaqueUnmeshedSvcSC  = "slow-cooker-opaque-unmeshed-svc"
+	tcpMetricRE          = regexp.MustCompile(
 		`tcp_open_total\{direction="inbound",peer="src",target_addr="[0-9\.]+:[0-9]+",tls="true",client_id="default\.linkerd-opaque-ports-test\.serviceaccount\.identity\.linkerd\.cluster\.local"\} [0-9]+`,
 	)
+	tcpMetricOutUnmeshedRE = regexp.MustCompile(
+		`tcp_open_total\{direction="outbound",peer="dst",authority="[a-zA-Z\-]+\.[a-zA-Z\-]+\.svc\.cluster\.local:[0-9]+",target_addr="[0-9\.]+:[0-9]+",tls="no_identity",no_tls_reason="not_provided_by_service_discovery",.*\} [0-9]+`,
+	)
 	httpRequestTotalMetricRE = regexp.MustCompile(
 		`request_total\{direction="outbound",authority="[a-zA-Z\-]+\.[a-zA-Z\-]+\.svc\.cluster\.local:8080",target_addr="[0-9\.]+:8080",tls="true",.*`,
 	)
+	httpRequestTotalUnmeshedRE = regexp.MustCompile(
+		`request_total\{direction="outbound",authority="[a-zA-Z\-]+\.[a-zA-Z\-]+\.svc\.cluster\.local:8080",target_addr="[0-9\.]+:8080",tls="no_identity",.*`,
```

Comment:
Why not check for the specific authority i.e `svc-opaque-unmeshed.linkerd-opaque-ports-test.svc.cluster.local:8080` as we know it wil lbe that for sure?

Reply:
No reason in particular, think at the time it felt more intuitive to me to keep it as any authority in case we might want to reuse it later. Since I doubt this will be the case in the near future I'll change it to be explicit 👍🏻

Reply:
Makes sense!